src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -170,6 +170,20 @@ const ResultsBlock = (): JSX.Element | null => {
     return null;
   }
 
+  if (
+    !crossModelQueryResults ||
+    Object.values(crossModelQueryResults).every((resultArray) =>
+      resultArray.every((result) => result === null)
+    )
```

Comment:
It could be nice to leave a comment here about why this is necessary.

Reply:
Done!